components/publish/PublishErrMessage.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+defineProps<{
+  describedby: string
```

Comment:
```suggestion
  text: string
```

Reply:
IDK if we change this name, other contributers will know this property is references `aria-describeby` 🤔

Reply:
Sure, resolved